web/docs/glossary.md

```diff
@@ -0,0 +1,170 @@
+If you are missing a term, please open a [GitHub Discussion][new-discussion] or
+ping us in our [Discord chat](/discord).
+:::
+
+[new-discussion]: https://github.com/orgs/tenzir/discussions/new?category=questions-answers
+
+## App
+
+Web user interface to access [platform](#platform) at
+[app.tenzir.com](https://app.tenzir.com).
+
+The app is a web application that partially runs in the user's browser. It is
+written in [Svelte](https://svelte.dev/).
+
+- [Use the app](./setup-guides/use-the-app/README.md)
+
+## Catalog
+
+Maintains [partition](#partition) ownership and metadata.
+
+The catalog is a component in the [node](#node) that owns the
+[partitions](#partition), keeps metadata about them, and maintains a set of
+sparse secondary indexes to identify relevant partitions for a given query. It
+offers a transactional interface for adding and removing partitions.
+
+- [Tune catalog
+  fragmentation](./setup-guides/tune-performance/README.md#tune-catalog-fragmentation)
+- [Configure catalog and partition indexes](./setup-guides/tune-performance/README.md#configure-catalog-and-partition-indexes)
+
+## Connector
+
+Manages chunks of raw bytes by interacting with a resource.
+
+A connector is either a *loader* that acquires bytes from a resource, or a
+*saver* that sends bytes to a resource. Example connectors are
+[`file`](./connectors/file.md), [`kafka`](./connectors/kafka.md), and
+[`nic`](./connectors/nic.md).
+
+- See all available [connectors](./connectors.md)
+
+## Format
+
+Translates between bytes and events.
+
+A format is either a *parser* that converts bytes to events, or a *printer*
+that converts events to bytes. Example formats are [`json`](./formats/json.md),
+[`cef`](./formats/cef), and [`pcap`](./formats/pcap.md).
+
+- See all available [formats](./formats.md)
+
+## Index
+
+Optional data structures for accelerating historical queries.
+
+Tenzir has *sparse* and *dense* indexes. Sparse indexes live in memory and point
+to [partitions](#partition), whereas dense indexes live within a partition and
+point to individual rows within the partition [store](#store).
+
+- [Configure catalog and partition indexes](./setup-guides/tune-performance/README.md#configure-catalog-and-partition-indexes)
+
+## Node
+
+A host for [pipelines](#pipeline) and storage reachable over the network.
+
+The `tenzir-node` binary starts a node in a dedicated server process that
+listens on TCP port 5158.
+
+- [Deploy a node](./setup-guides/deploy-a-node/README.md)
+- [Use the app](./setup-guides/use-the-app/README.md) to manage a node
+- Use the [REST API](./rest-api.md) to manage a node.
+- [Import into a node](./user-guides/import-into-a-node.md)
+- [Export from a node](./user-guides/export-from-a-node.md)
+
+## Metrics
+
+Runtime statistics about pipeline and node execution.
+
+- [Collect metrics](./setup-guides/collect-metrics.md)
+- See all available [metrics](./metrics.md) details
+
+## Operator
+
+The atomic building block of a [pipeline](#pipeline).
```

Comment:
The atomic is not necessarily true here, as every pipeline itself is also a valid operator again. I would just remove the word.